editor/js/Config.js

```diff
@@ -3,10 +3,25 @@ function Config() {
 	const name = 'threejs-editor';
 
 	const userLanguage = navigator.language.split( '-' )[ 0 ];
-
 	const suggestedLanguage = [ 'fr', 'ja', 'zh' ].includes( userLanguage ) ? userLanguage : 'en';
 
+	function getSidebarMinWidth( language = suggestedLanguage ) {
+
+		return { // >= #sidebar min-width (335px)
+
+			'en': 350,
+			'fr': 355,
+			'ja': 395,
+			'zh': 335
```

Comment:
This is yet more code that needs to be maintained.

I'm sure there's a solution for this issue that doesn't create more maintenance work.

Reply:
css.

Reply:
ok, i found a way to eliminate those numbers, by auto sniffing sum of tabs `offsetWidth` of the widest tabbed panel